projects/kit/components/input-files/input-files.directive.ts

```diff
@@ -40,14 +40,14 @@ export class TuiInputFilesDirective {
         return this.el.nativeElement.multiple ?? this.options.multiple;
     }
 
-    @HostBinding('capture')
+    @HostBinding('attr.capture')
     get capture(): TuiInputFilesOptions['capture'] {
         return (
```

Comment:
Maybe this is better?
```ts
    @HostBinding('attr.capture')
    get capture(): string | null {
        return this.el.nativeElement.getAttribute('capture') || this.options.capture;
    }
```
Since we cannot really rely on the `property` due to platform specifics and we bind an `attribute` anyway.

Reply:
fixed